src/app/pages/administration/url-checker/url-checker.jsx

```diff
@@ -0,0 +1,336 @@
+import React, { useContext, useEffect, useState } from 'react';
+import {
+  Box,
+  Button,
+  Card,
+  CardContent,
+  CircularProgress,
+  Container,
+  Typography,
+  makeStyles,
+} from '@material-ui/core';
+import { useSnackbar } from 'notistack';
+import { useFormik } from 'formik';
+import * as yup from 'yup';
+import CancelIcon from '@material-ui/icons/Cancel';
+import CheckCircleIcon from '@material-ui/icons/CheckCircle';
+import VerifiedUserIcon from '@material-ui/icons/VerifiedUser';
+
+import { ChplTextField } from 'components/util';
+import { utilStyles, palette } from 'themes';
+import usePostUrlChecker from 'api/url-checker';
+import { UserContext } from 'shared/contexts';
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
```suggestion
const useStyles = makeStyles({
```

We've been moving away from the function use on makestyles, unless necessary (which I've not seen as necessary anywhere yet). That said, even with the function definition this makeStyles is using a `theme` variable that's not defined anywhere. Even in the `const classes...` line. If the `theme` variable is needed, it should be imported in the `import { utilStyles, pallete}...` line, and then it would be available in the style definition section